subprojects/internal-testing/src/main/groovy/org/gradle/util/TestPrecondition.groovy

```diff
@@ -154,6 +154,9 @@ enum TestPrecondition implements org.gradle.internal.Factory<Boolean> {
     }),
     KOTLIN_SCRIPT({
         JDK8_OR_LATER.fulfilled && FIX_TO_WORK_ON_JAVA9.fulfilled && NOT_JDK_IBM.fulfilled
+    }),
+    XCODE({
+        MAC_OS_X.fulfilled
```

Comment:
Should this also be based on a particular version of XCode or even Xcode being installed?

Reply:
I will leave it like that for now. I added a comment about it.